Tests/Marketplace/upload_packs.py

```diff
@@ -473,7 +492,8 @@ def main():
             pack.cleanup()
             continue
 
-        task_status = update_index_folder(index_folder_path=index_folder_path, pack_name=pack.name, pack_path=pack.path)
+        task_status = update_index_folder(index_folder_path=index_folder_path, pack_name=pack.name, pack_path=pack.path,
```

Comment:
I think after all done, we can add `pack.cleanup()` after line 502 (pack.status = PackStatus.SUCCESS.name), just to clear unnecessary pack folder